features/product/viewing_products/viewing_products_from_specific_taxon.feature

```diff
@@ -5,21 +5,20 @@ Feature: Viewing products from a specific taxon
     I want to be able to view products from a specific taxon
 
     Background:
-        Given the store has currency "Euro"
```

Comment:
why is this removed?

Reply:
this step is unnecessary in these scenarios. so I deleted it :)